docs/modules/development/pages/rest/meta-data.adoc

```diff
@@ -2,9 +2,17 @@
 [[metadata-rest]]
 = Metadata
 
-You can query the actual metadata of nodes, interfaces, and services via REST endpoints.
+You can query, add, modify, or delete the actual metadata of nodes, interfaces, and services via REST endpoints.
 
-== GETs (reading metadata)
+Modification of metadata entries is limited to user defined contexts (starting with `X-`).
+Read about xref:operation:deep-dive/admin/metadata.adoc#metadata-contexts[how Metadata context work] for further details.
+
+WARNING: It is highly recommended to not modify any metadata using this interface but use the provided requisition mechanisms instead.
+Altering metadata from a node configured in a requisition will change your existing metadata on every synchronisation of the originating requisition in an unpredicted way.
+This can lead to data-loss. You have been warned!
```

Comment:
```suggestion
Nodes provisioned via requisition may have metadata overwritten during the resynchronization process.
This can lead to loss of custom metadata defined outside the requisition.
```